src/cluster_legacy.h

```diff
@@ -405,6 +405,7 @@ struct clusterState {
     int failover_auth_rank;           /* This replica rank for current auth request. */
     int failover_failed_primary_rank; /* The rank of this instance in the context of all failed primary list. */
     uint64_t failover_auth_epoch;     /* Epoch of the current election. */
+    uint64_t failover_config_epoch;   /* Epoch of the current primary config epoch. */
```

Comment:
Repeated word epoch. Maybe like this?

```suggestion
    uint64_t failover_config_epoch;   /* Config epoch of the current primary. */
```